lib/archethic/mining/distributed_workflow.ex

```diff
@@ -764,14 +765,15 @@ defmodule Archethic.Mining.DistributedWorkflow do
         :info,
         {:replication_error, reason},
         :replication,
-        _ = %{context: %ValidationContext{transaction: tx}}
+        data = %{context: %ValidationContext{transaction: tx}}
       ) do
     Logger.error("Replication error - #{inspect(reason)}",
       transaction_address: Base.encode16(tx.address),
       transaction_type: tx.type
     )
 
-    :stop
+    {:next_state, {:return_error, "replication_error"}, data}
```

Comment:
You should take the reason of the error as input to notify back to the welcome node